### ISSUE TITLE
Progress status

### ISSUE THREAD

**Issue by redphx** [redphx/joydance-next#1]
1. Checking JD 2023 app.

**Comment by ghost** [comment]
btw if u need me to test anything, lmk, I have pre-ordered the game on xbox and switch.

**Comment by ghost** [comment]
I got the game early, If u need early testing lmk! I was able to get the game 4 days early on switch :D

**Comment by redphx** [comment]
@RockyDaNerd their new app is completely different from the old one, so it'll take a lot more time to work with.

**Comment by ghost** [comment]
yeah no worries, take ur time! its no rush, just wanted to let u know that I already have access to the game if u need me to do some early testing (thats if u dont want to buy the game yet)

**Comment by Rdrigo98** [comment]
any updates?

**Comment by oakleyplays** [comment]
updates? @redphx

**Comment by JunioSplat** [comment]
Please, feedback us <3

**Comment by oakleyplays** [comment]
@redphx 
just dance 2024 is out, do we have any updates??

**Comment by Matty8836** [comment]
Any updates for support on jd23/24 or is there already support? @redphx

**Comment by BroJac5246** [comment]
> Any updates for support on jd23/24 or is there already support? @redphx 

As mentioned above, the new app uses a different protocol so there isn't support yet (ever?). If support is added, it will almost certainly be as a new release, so I'd set your notification settings for this repo alert you of those. That said, redphx hasn't been active here for a long time so support will probably come via a fork if someone ever does that.

**Comment by redphx** [comment]
Sorry for the lack of updates.  
I don't have time at the moment to work on this project. But the last time I checked, the new games use a totally different protocol so the old `joydance` app won't compatible with these new games. Also, they obfuscate the app so it's much more difficult to reverse-engineer this time.  
Sorry for the disappointing news.

**Comment by oakleyplays** [comment]
> Sorry for the lack of updates. I don't have time at the moment to work on this project. But the last time I checked, the new games use a totally different protocol so the old `joydance` app won't compatible with these new games. Also, they obfuscate the app so it's much more difficult to reverse-engineer this time. Sorry for the disappointing news.

its fine, just in like 4-5 months maybe 2025 might be out 👀
